autogen.sh

```diff
@@ -94,6 +94,46 @@ for file in `ls $BASE_DIR/src/sst/elements`; do
 	fi
 done
 
+for file in `ls $BASE_DIR/src/sst/elements/osseous/examples/`; do
```

Comment:
@gvoskuilen @nmhamster Someone more familiar with autogen want to comment on this?

Reply:
The idea here was to include osseous examples as the part of central build. Osseous is part of rest of the elements. If rest of the examples build's then osseous will also build. But examples within it were not included while this build process. So I had to include this code as part of autogen.

Reply:
Autogen updates retrieved in 84416d6